pkg/gatherers/workloads/gather_workloads_info.go

```diff
@@ -41,6 +41,13 @@ const (
 	podsLimit = 8000
 )
 
+// keys are namespace / pod / containerID and the final value is the workloadRuntimeInfoContainer for this container
+type workloadRuntimes map[string]map[string]map[string]workloadRuntimeInfoContainer
```

Comment:
perhaps we could build a new type encompassing the strings for `namespace`, `pod` and `containerID` and then use it as the map key?